src/test/regress/sql/cpu_priority.sql

```diff
@@ -71,7 +71,9 @@ SET citus.cpu_priority_for_logical_replication_senders = 15;
 SELECT master_move_shard_placement(11568900, 'localhost', :worker_2_port, 'localhost', :worker_1_port, 'force_logical');
 SET citus.max_high_priority_background_processes = 3;
 SELECT master_move_shard_placement(11568900, 'localhost', :worker_1_port, 'localhost', :worker_2_port, 'force_logical');
-CALL citus_cleanup_orphaned_shards();
+SET client_min_messages TO ERROR;
```

Comment:
Why are these necessary?  I thought wait_for_resource_cleanup already removed logs. i.e. what errors is this showing?

(same for these same statements in other tests files)

Reply:
not errors but warnings like "cleanup failed for 1 object out of 4"

Reply:
Hmm, okay. Any reason wait_for_resource_cleanup  shows warnings? Sounds like it would be better to hide those also.